examples/efficient_net.py

```
@@ -0,0 +1,298 @@
+        x = layers.Conv3D(filters_out, 1,
+                        padding='same',
+                        use_bias=False,
+                        kernel_initializer=CONV_KERNEL_INITIALIZER,
+                        name=name + 'project_conv')(x)
+        x = layers.BatchNormalization(axis=bn_axis, name=name + 'project_bn')(x)
+
+        if (id_skip is True and strides == 1 and filters_in == filters_out):
+            if drop_rate > 0:
+                x = layers.Dropout(drop_rate,
+                                noise_shape=None,
+                                name=name + 'drop')(x)
+            x = layers.add([x, inputs], name=name + 'add')
+
+        return x
+
+
+    def call(self, inputs: tf.Tensor, training=None, mask=None) -> tf.Tensor:
+        """
+        Builds graph based on built layers.
+
+        :param inputs: shape = (batch, f_dim1, f_dim2, f_dim3, in_channels)
+        :param training:
+        :param mask:
+        :return: shape = (batch, f_dim1, f_dim2, f_dim3, out_channels)
+        """
+        img_input = layers.Input(tensor=inputs, shape=self.image_size)
+        bn_axis = 4  
+        # Build stem
+        x = img_input
+        # x = layers.ZeroPadding3D(padding=self.correct_pad(x, 3),
+        #                         name='stem_conv_pad')(x)
+
+        x = layers.Conv3D(self.round_filters(32), 3,
+                        strides=2,
+                        padding='same',
+                        use_bias=False,
+                        kernel_initializer=CONV_KERNEL_INITIALIZER,
+                        name='stem_conv')(x)
+        x = layers.BatchNormalization(axis=bn_axis, name='stem_bn')(x)
+        x = layers.Activation(self.activation_fn, name='stem_activation')(x)
+        blocks_args = deepcopy(DEFAULT_BLOCKS_ARGS)
+
+        b = 0
+        # 计算总的block的数量
+        blocks = float(sum(args['repeats'] for args in blocks_args))
+        for (i, args) in enumerate(blocks_args):
+            assert args['repeats'] > 0
```

please raise an error if `args["repeats"]<=0`